package/pyinstaller/create-installer.iss.template

```diff
@@ -26,7 +26,8 @@ Source: "dist\silx\*"; DestDir: "{app}"; Flags: ignoreversion recursesubdirs
 Source: "silx.ico"; DestDir: "{app}"
 
 [Icons]
-Name: "{group}\silx"; Filename: "{app}\silx-view.exe"; IconFilename: "{app}\silx.ico"
+Name: "{group}\silx view"; Filename: "{app}\silx.exe"; Parameters: "view"; IconFilename: "{app}\silx.ico"
+Name: "{group}\silx compare"; Filename: "{app}\silx.exe"; Parameters: "compare"; IconFilename: "{app}\silx.ico"

```

Comment:
Just checked, the images to compare needs to be passed to the command line, there is no menu to open them from the GUI, so no interest for now to expose `silx compare` as an icon.
I would remove for now it but keep the `silx` group.

```suggestion

```